.github/workflows/rc.yml

```diff
@@ -1,9 +1,12 @@
 name: rc
 
 on:
-  push:
-    branches:
-      - 'v[0-9]+.*'
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "version such as '3.1.0'"
+        required: true
+        default: '3.1.0'
```

Comment:
hardcode 3.1.0?

Reply:
prefer to do not use default, if not provide version, it would be errors

Reply:
make sense!